package.json

```diff
@@ -28,6 +28,7 @@
     "@hylo/shared": "workspace:*"
   },
   "devDependencies": {
+    "standard": "^17.1.2",
```

Comment:
This, and the removals of standard from the individual packages is what to pay attention to here. Everything else has no functional impact.